airdrops/index_v2.json

```diff
@@ -155,6 +161,7 @@
             "icon": "starknet.svg",
```

Comment:
The myso finance airdrop seems to not have an entry and afaik it's **not** decoded.